components/dependency_injection/compilation.rst

```diff
@@ -425,8 +425,8 @@ been run, use::
 
 You can also control the order in which compiler passes are run for each
 compilation phase. Use the optional third argument of ``addCompilerPass()`` to
-set the priority as an integer number. The default priority is ``0`` and the higher
-its value, the earlier it's executed::
+set the priority as an integer number. The default priority is ``0`` and the smaller
+the value is, the earlier it's executed::
```

Comment:
This does not look right. Higher priorities mean earlier execution.

Reply:
Yes sorry, you can forget this PR, you have right